pages/ButtonsPage.ts

```diff
@@ -0,0 +1,41 @@
+import { Page } from 'playwright';
+
+export class ButtonsPage {
+  private page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  async goToButtonsPage() {
+    await this.page.goto('/buttons');
+  }
+
+  async isClickMeMessageVisible(expectedMessage: string) {
+    await this.page.waitForSelector(`#dynamicClickMessage:has-text("${expectedMessage}")`);
+    const locator = this.page.locator(`#dynamicClickMessage:has-text("${expectedMessage}")`);
```

Comment:
This locator should be inside the constructor. It will look the same as page declaration. 
```suggestion
    return this.yourLocator.isVisible()
```